docs/samples/browser-plugin-meetings/app.js

```diff
@@ -956,36 +955,6 @@ const localMedia = {
   }
 }
 
-function handleTrackUnpublished(track, mediaElement, debugString) {
-  track.on('published-state-update', ({isPublished}) => {
```

Comment:
@brycetham - To avoid confusion with Todo, is it okay if I add your example commit to the ticket and ensure all the scenarios are handled based on your deleted changes?

If you still think this needs to be kept, I can go ahead and make changes for it.

Reply:
It's easier to add later because this commit is mostly changes cherry-picked from yours. I'll add it later since you're okay too. Thanks a lot 😊